andela/suyabay
# Bg fix user editprofile 121839485

#### What does this PR do?
- Align the user profile edit page properly
#### Description of Task to be completed?
- Remove unnecessary row class declaration
- Align update username form and avatar form side by side.
- Style the form headers and give it a class selector and reduce the font size to 18px.
#### How should this be manually tested?
-  User should log in and click on profie  or visit `https://suyabay-staging.herokuapp.com/profile/edit`. The authenticated user should see a well aligned user profile page.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)

![screen shot 2016-06-20 at 11 56 10 am](https://cloud.githubusercontent.com/assets/14159435/16191828/0cf3a95e-36de-11e6-9e31-8d2f7236e33a.png)
![screen shot 2016-06-20 at 11 55 32 am](https://cloud.githubusercontent.com/assets/14159435/16191829/0cf4fc32-36de-11e6-8f82-f2f8b2ad0b23.png)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
